models/engine/file_storage.py

```diff
@@ -68,3 +68,38 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Get a single object from __objects
+
+        Args:
+            cls     - string representing the class name
+            id      - string representing the object ID
+
+        Returns:
+            Object base on the class and id or else None.
+        """
+        if not cls or not id:
+            return None
+        if type(cls) is not str:
+            cls = cls.__name__
+        key = '{}.{}'.format(cls, id)
+        if key in self.__objects:
```

Comment:
How about one-lining this with the `get` method - `return self.__objects.get(key, None)`